regression/cbmc-java/inherited_static_field3/test.java

```diff
@@ -0,0 +1,15 @@
+public class test extends parent {
+
+  public static void main() {
+
+    assert x == parent.x;
+
+  }
+
+}
+
+class parent {
```

Comment:
To keep consistent with other tests - consider calling this `OpaqueParent` rather than `parent`